src/legacy/ui/public/courier/fetch/components/shard_failure_open_modal_button.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+// @ts-ignore
+import { npStart } from 'ui/new_platform';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { EuiButton } from '@elastic/eui';
+
+import { ShardFailureModal } from './shard_failure_modal';
+import { ResponseWithShardFailure, Request } from './shard_failure_types';
+
+interface Props {
+  request: Request;
+  response: ResponseWithShardFailure;
+  title: string;
+}
+
+export function ShardFailureOpenModalButton({ request, response, title }: Props) {
+  function onClick() {
+    const modal = npStart.core.overlays.openModal(
+      <ShardFailureModal
+        request={request}
+        response={response}
+        title={title}
+        onClose={() => modal.close()}
+      />,
+      {
+        className: 'shardFailureModal',
+      }
+    );
+  }
+  return (
+    <EuiButton onClick={onClick} data-test-subj="openShardFailureModalBtn">
```

Comment:
Can you change this to an `EuiButtonEmpty flush="left"` and match the color of the toast with `warning`?

Reply:
EuiButtonEmpty allows only the following colors:

Type '"warning"' is not assignable to type '"text" | "disabled" | "primary" | "danger" | "ghost" | undefined'.ts(2322)

I kept EuiButton for now, setting it to warning:
<img width="364" alt="Bildschirmfoto 2019-07-25 um 07 21 04" src="https://user-images.githubusercontent.com/463851/61847998-dd2f8e00-aeac-11e9-83a7-acd671ae655b.png">

Reply:
That works.